src/commands/utils/configureWorkflowHelper.ts

```diff
@@ -17,8 +17,8 @@ export function configureStarterConfigDataForAKS(
       // Load vscode resoruce yaml file and replace content.
     const yamlPathOnDisk = vscode.Uri.file(path.join(extensionPath.result, 'resources', 'yaml', `${workflowName}.yml`));
     const starterFileAutoFillContent = fs.readFileSync(yamlPathOnDisk.fsPath, 'utf8')
-                                            .replace('<RESOURCE_GROUP>', resourceName)
-                                            .replace('<CLUSTER_NAME>', clusterName);
+                                            .replace('your-resource-group', resourceName)
```

Comment:
💡 there will be no scenario where this could defer between env var or the deployment level use, if yes then I would recommend change the placeholder in current exiting deployment files.

Reply:
Correct! This should work.